draft-ietf-rats-corim.md

```diff
@@ -1599,11 +1599,11 @@ The ACS must maintain the authority information for each EMT. There can be
 multiple entries in `state-triples` which have the same `environment-map`
 and a different `authorized-by` field (see {{sec-authorized-by}}).
 
-If the merged measurement-value-map contains duplicate codepoints and the
+If the merged `measurement-values-map` contains duplicate codepoints and the
```

Comment:
I'm wondering if this case can never happen given it follows cddl. CDDL shouldn't let you have `state-triples` where the `measurement-values-map` contains duplicate code points. I wonder if we really need to describe "merging" since the case above describes redundant copies of an EMT.  If so, you can just delete redundant duplicate EMTs. No need to describe deeper "merging" semantics.